### ISSUE TITLE
width stopped working

### ISSUE THREAD

**Issue by kleinash** [JuliaGeometry/GeometryTypes.jl#43]
Hi,

Routine width() stopped working. I found out that its now widths, but havent seen anything documenting the change - am I missing it somewhere? 

Thx

**Comment by sjkelly** [comment]
Yes, we messed up semver on releases and 0.2.1 is breaking and supposed to
be 0.3.0. Sorry about that.

`width` is deprecated in favor of `widths` which returns a vector of all
lengths rather than a scalar. This means we don't have to use the ambiguous
`length` and `height` functions for the other axes.

**Comment by kleinash** [comment]
I get it - knew there would be a valid logical explanation. Closed the issue cause of the other thread.. I'll stick ref to your answer there.

**Comment by sjkelly** [comment]
Just saw that one. Thanks!
